src/components/Common/GoBack/GoBack.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { IoReturnUpBackSharp } from 'react-icons/io5';
+import { useNavigate } from 'react-router-dom';
+
+import styles from 'components/Common/GoBack/GoBack.module.scss';
+
+const GoBack = () => {
+  const navigate = useNavigate();
+
+  return (
+    <div className={styles.goHomeWrapper}>
+      <div className={styles.icons}>
```

Comment:
icon

Reply:
this 'div' now is not needed, i remove his